packages/dynamite/dynamite_runtime/lib/src/dynamite_client.dart

```diff
@@ -404,14 +404,14 @@ class DynamiteClient {
   /// The first one matching the required authentication type will be used.
   final List<DynamiteAuthentication> authentications;
 
-  /// Makes a request against a given [path].
-  Future<HttpClientResponse> doRequest(
+  /// Makes a request against a given [path] using the [baseURL] and [baseHeaders].
+  Future<HttpClientResponse> executeRequestFromBaseValues(
```

Comment:
I dislike the name of this function.
Maybe we should continue the theme of `executeRequest` and `executeRawRequest` although that's also not perfect